.circleci/detect-functional-changes.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+VERSION_CHANGE_TRIGGERS="setup.py MANIFEST.in openfisca_country_template"
+
+if git diff-index --quiet origin/master -- $VERSION_CHANGE_TRIGGERS ":(exclude)*.md"
```

Comment:
@fpagnoux Beware, this is not the same check as it was before I factored it: please double-check based on https://github.com/openfisca/country-template/pull/41#discussion_r188825589 that this is the proper check we want to apply here!

Reply:
See reply on the said comment